packages/enzyme-adapter-utils/src/Utils.js

```diff
@@ -95,6 +95,15 @@ export function nodeTypeFromType(type) {
   return 'function';
 }
 
+export function isIterable(obj) {
+  return (
+    typeof Symbol === 'function' &&
+    typeof Symbol.iterator === 'symbol' &&
+    obj != null &&
```

Comment:
i'd probably move the `obj` check higher, since that's faster then checking Symbols

Reply:
Awesome, I wasn't aware of that! Thanks!